storybook/storybook-react/src/Dialog/Dialog.docs.mdx

```diff
@@ -0,0 +1,19 @@
+import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as DialogStories from "./Dialog.stories.tsx";
+import README from "../../../../packages/css/src/dialog/README.md?raw";
+
+<Meta of={DialogStories} />
+
+<Markdown>{README}</Markdown>
+
+<Primary />
+
+<Controls />
+
+## Scroll Content
+
+<Canvas of={DialogStories.ScrollContent} className="amsterdam-dialog-story" />
+
+## No Actions
+
+<Canvas of={DialogStories.NoActions} className="amsterdam-dialog-story" />
```

Comment:
`DialogStories.ShowDialog` is missing here

Reply:
There was an issue with the `showModal()` function causing the button not to open the dialog, I've now set an `id` to the modal but typescript interferes with the queryselector forcing me to cast the element to any:
```
document.querySelector('#showdialog' as any)?.showModal()
```